addons/resource.language.en_gb/resources/strings.po

```diff
@@ -18232,3 +18232,8 @@ msgstr ""
 msgctxt "#38030"
 msgid "This option uses frame-packing to output full resolution for 3D through HDMI.[CR]Enabling this improves quality of Multiview Video Coding (MVC) videos, but may not be supported by all displays."
 msgstr ""
+
+#: xbmc\video\dialogs\GUIDialogVideoSettings.cpp
```

Comment:
Mind adding a contextual comment so that people (devs and translators) know in which context it's used? Something like: